testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ContactListAdapter.kt

```diff
@@ -19,18 +21,30 @@ class ContactListAdapter : RecyclerView.Adapter<ContactListAdapter.ViewHolder?>(
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        holder.contactName.text = contactEntries[position]
+        val contact = contactEntries[position]
+        holder.contactId.text = contact.id
+        bindNonVisibleView("Name: ", contact.name, holder.contactName)
+        bindNonVisibleView("Email: ", contact.email, holder.contactEmail)
+
         holder.contactRemoveButton.setOnClickListener { removeContactAt(position) }
     }
 
+    private fun bindNonVisibleView(prefix: String, text: String?, holderView: TextView) {
```

Comment:
I think, we can just write `bindView` ? Because this function is making the views "visible" / "gone", so "NonVisible" keyword is making a confusion to me... 🤔